.g8/checkboxPage/app/views/$className$View.scala.html

```diff
@@ -2,19 +2,18 @@
 @import models.{Mode, $className$}
 
 @this(
-    main_template: MainTemplate,
-    formHelper: FormWithCSRF
+layout: template.Layout,
+formHelper: FormWithCSRF,
 )
 
 @(form: Form[Set[$className$]], mode: Mode)(implicit request: Request[_], messages: Messages)
 
-@main_template(
-    title = s"\${errorPrefix(form)} \${messages("$className;format="decap"$.title")}"
+@layout(
```

Comment:
These g8 template views need looking at again. They are still trying to use old components that don't exist anymore - i.e. in this file I can see `components.input_checkboxes`, `components.error_summary`, `components.submit_button`. These are referring to the old asset component files that have been deleted in this PR, so if this template was used to generate an actual view it wouldn't compile.

Once again I have a lack of knowledge when it comes to g8 templates, and not sure how likely it is we would need to use them seeing as we're the live services team, but if they are in the service let's try to maintain them. If there's difficulty in doing this then we can discuss if removal is better.

Reply:
Removed g8 folder